src/timedata.cpp

```diff
@@ -15,9 +15,8 @@
 #include <util/translation.h>
 #include <warnings.h>
 
-
-static RecursiveMutex cs_nTimeOffset;
-static int64_t nTimeOffset GUARDED_BY(cs_nTimeOffset) = 0;
+static Mutex g_timeoffset_mutex;
+static int64_t nTimeOffset GUARDED_BY(g_timeoffset_mutex) = 0;
```

Comment:
nit: `s/nTimeOffset/g_time_offset/`, feel free to ignore. I just wonder - you changed the name of one global variable why not the other too?

Reply:
I hope this commit is mutex related only :)